### DISCUSSION TITLE
Let Github PullRequests detect base branch automatically

### DISCUSSION THREAD

**Discussion by HaimCandiTech** [community/community#37480] [Pull Requests]
Hey GitHub ❤️ 
Usually when I branch out from another branch, there will be a time when I'll want to merge it to the base branch
I usually do it using a Pull Request.
Sadly when creating a Pull Request, the base branch for the PR is default as master/main
Any chance there is some configuration to enable it to detect the branch that we branched from as the base branch?
Thanks in advance

**Comment by ossanna16** [comment]
Hi there @HaimCandiTech and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
